### DISCUSSION TITLE
css bug

### DISCUSSION THREAD

**Discussion by AlexVagrant** [community/community#13242] [Other Feature Feedback, Questions, & Ideas]
<img width="896" alt="image" src="https://user-images.githubusercontent.com/25078327/159462916-eefec04a-4d0a-4d31-be42-cae36e366599.png">

**Comment by AlexVagrant** [comment]
content: 精通rust/csharp/delphi/c/c++/java/python/ruby/erlang/golang/vb/的拼写

**Comment by jamieshark** [comment]
Thanks! We've filed this as a bug and will work on a fix soon.

**Comment by basitmir7** [comment]
Well the problem here is that the parent container i.e the card in your case has a specific height and is not able contain the text ,according to me you can change the property to "min-height" instead of "height" . I am sure this will help.

**Comment by ApidriuC** [comment]
You can fix the problem with overflow: hidden and you must take into account the size of the text that goes inside the container

**Comment by alikarakoc** [comment]
Hi,
 
`<div>精通rust/csharp/delphi/c/c++/java/python/ruby/erlang/golang/vb/的拼写</div>
`
```
 div {
    width: 60px;
    overflow: hidden; /* hide overflows */
    white-space: nowrap; /* don't go to the bottom line */
    text-overflow: ellipsis; /* if it is long put three dots */
}
```
This will eliminate the problem. Please try.

If it works for you please don't forget to mark it as correct.
I am thankful to you.

**Comment by dendcode** [comment]
I would use flex and padding and overflow

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report

**Comment by mrtuvn** [comment]
min-width: 0 at child flex item may help
